livre/parts/header.html

```diff
@@ -4,5 +4,5 @@
 <!-- wp:page-list {"isNavigationChild":true,"showSubmenuIcon":true,"openSubmenusOnClick":false} /-->
 <!-- /wp:navigation -->
 
-<!-- wp:site-title {"style":{"typography":{"fontStyle":"italic","fontWeight":"400"}}} /--></div>
-<!-- /wp:group -->
\ No newline at end of file
+<!-- wp:site-title {"style":{"spacing":{"padding":{"top":"0.35ch"}},"typography":{"textTransform":"uppercase"}}} /--></div>
```

Comment:
The padding here is a little weird, but is necessary to ensure that the text properly aligns vertically next to the menu button, since there are no descenders when the text is all caps.